website/Twig/AsyncAwsTwigExtension.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Website;
+
+use Twig\Extension\AbstractExtension;
+
+class AsyncAwsTwigExtension extends AbstractExtension
+{
+    /**
+     * @var array|null
+     */
+    private $manifestCache;
+
+    /**
+     * @var array|null
+     */
+    private $entrypointsCache;
+
+    public function getFunctions()
+    {
+        return [
+            new \Twig_SimpleFunction('asset', [$this, 'getAssetUrl']),
+            new \Twig_SimpleFunction('renderStyle', [$this, 'renderStyle'], [
```

Comment:
use `\Twig\TwigFunction` to be foward-compatible with new Twig versions (Twig 3 uses only namespaced classes)

Reply:
Couscous is using Twig 1. Is there a forward bridge from Twig 1 so I can use namespaces?

Reply:
Yes, it does. https://github.com/async-aws/aws/pull/427

Reply:
thus, you already use the namespace for the AbstractExtension base class :smile: